packages/translations/scripts/extract-string.js

```diff
@@ -6,8 +6,9 @@ const getRegexPattern = () => /(i18n_default_text={?|localize\()\s*(['"])\s*(.*?
 
 const getStringsFromInput = (input, i18n_marker = getRegexPattern()) => {
     const messages = [];
+    const result = i18n_marker.exec(input);
 
-    while ((result = i18n_marker.exec(input)) !== null) {
```

Comment:
as we checked with @iman-fs  we need to assign the result each time that while loop is running.
@iman-fs would you please share your idea here.

Reply:
```
You can change it like this:      
    let runLoop = true;
    while (runLoop) {
        const result = i18n_marker.exec(input);
        if (result == null) runLoop = false;
        const extracted = result[3];
        // Replace escape characters.
        messages.push(extracted.replace(/\\/g, ''));
    }
```

Reply:
since Eslint [no-constant-condition](https://eslint.org/docs/latest/rules/no-constant-condition) rule disallows using 
    `while(true)` and then just break the loop if `result==null` , you have to write it like above code I think